README.md

```diff
@@ -1,20 +1,69 @@
-- [Supervisor](https://github.com/livechat/sample-apps/tree/master/supervisor) - widget for helping you monitor weekly progress of your agents and also their availability.
+The statistics include the following information:
 
-In the source code, you can find the code for **Authorization** and **data fetching** from our server.
+- **Agent working time** 
+- **Chatting time** 
+- **Agent ratings** 
 
-## Feedback
-If you find some bugs or have trobules implementing the code on your own, please create an issue on this repo.
 
-## If you're new to LiveChat
+## Preview
 
-**LiveChat** is an online customer service software with live support, help desk software, and web analytics capabilities. It's used by more than 27,000 companies all over the world. For more info, check out [LiveChat for Developers](https://developers.livechatinc.com/).
+![Alt Text](https://i.ibb.co/rfnHsTz/supervisor.png)
 
+# App setup
 
+## Before you start
+
+To use this application in your LiveChat dashboard, you'll need to create your own app in [Developer Console](https://developers.livechatinc.com/console) and get the **Client Id**.
+
+## Getting started
+
+1. Go to [Apps](https://developers.livechatinc.com/console/apps) in [Developer Console](https://developers.livechatinc.com/console).
+2. Click **New App** and give it an **App Name**.
+3. Choose the **LiveChat** product as the product you want to build for.
+4. Go to **Building Blocks**.
+5. Add **App Authorization** and mark it as *JavaScript App*. Your **Client Id** will be displayed there.
+6. Fetch the **Supervisor** app repository.
+7. In the app directory, do the following steps :
+
+    * Install dependencies (`npm install`).
+    * In your project, go to `src/utils/congif.js` and replace `client_id` with your own **Client Id** (the one from **Step 5**).
+    * Run your app (`npm start`).
+8. Add your app url (for example: `https://localhost:3000`) in these two locations:
+
+    * **Redirect URI whitelist**
+    * **Agent App Widgets**
+9. In **Private installation**, click **Install app**. 
+
+You should now be able to use **Supervisor** with LiveChat.
+
+# How it works
+
+[Agent App Widgets](https://developers.livechat.com/docs/extending-agent-app) are web applications loaded inside the LiveChat Agent App. All agents can interact with the widget during chats with customers. The widget itself is displayed in the Agent’s App sidebar.
+
+To get information such as tags and canned responses, you need to use [LiveChat Rest API](https://developers.livechat.com/docs/management/configuration-api/v2.0/).
+
+In order to pull data from our server, Rest API requires you to include an **access_token** in all the requests. You can get it using [Personal Access Tokens](https://developers.livechat.com/docs/authorization/agent-authorization#personal-access-tokens) and [JavaScript Widget API](https://docs.livechatinc.com/agent-app-widgets/#javascript-api).
```

Comment:
changed [LiveChat Boilerplate](https://docs.livechatinc.com/boilerplate/) for [Personal Access Tokens](https://developers.livechat.com/docs/authorization/agent-authorization#personal-access-tokens) as I believe that is the correct one now.